sacred/randomness.py

```diff
@@ -12,22 +13,29 @@
 def get_seed(rnd=None):
     if rnd is None:
         return random.randint(*SEEDRANGE)
-    return rnd.randint(*SEEDRANGE)
+    elif type(rnd) == opt.np.random.Generator:
+        return rnd.integers(*SEEDRANGE, dtype=int)
+    else:
+        return rnd.randint(*SEEDRANGE)
 
 
 def create_rnd(seed):
     assert isinstance(seed, int), "Seed has to be integer but was {} {}".format(
         repr(seed), type(seed)
     )
     if opt.has_numpy:
-        return opt.np.random.RandomState(seed)
+        if version.parse(opt.np.__version__) >= version.parse('1.19'):
```

Comment:
Can you run black to reformat the code? (Here: use double quotes)

Reply:
Done.